Jenkinsfile

```diff
@@ -24,6 +28,27 @@ pipeline {
                 withCredentials([usernamePassword(credentialsId: params.REGISTRY_URL, passwordVariable: 'TOKEN', usernameVariable: 'USERNAME')]) {
                     sh('. $WORKSPACE/bin/miniconda3/etc/profile.d/conda.sh; conda activate candig; echo ${TOKEN} | docker login ${REGISTRY_URL} -u ${USERNAME} --password-stdin; make docker-push OVERRIDE_REGISTRY=${REGISTRY_URL}')
                 }
+        stage('Build htsgetapp_server') {
+            steps {
+                sh '''
+                  cd ./lib/htsget-server/htsget_app
+                  ls -la
+                  bash htsget.sh
+                   '''
+                }
+            }
+        }
+      post {
+  //  Status message gets sent to the Slack Channel after the job finishes
+        always {
+          success {
+            slackSend channel: 'automation-email', message:"Build deployed successfully" -${currentBuild.currentResult} ${env.JOB_NAME} ${env.BUILD_NUMBER} ${env.BUILD_URL}
```

Comment:
I think the failure notification is useful, but receiving daily emails (based on the cron job cycle) for successful build may be too noisy.

Reply:
Yes @shaikh-rashid that make sense but it is only sending to the automation channel. I think it will be meaningful so we can click on the link provided and see the results in the output.